pkg/kv/dist_sender.go

```diff
@@ -1143,6 +1142,8 @@ func (ds *DistSender) sendToReplicas(
 	}
 	transport.SendNext(ctx, done)
 
+	notLeaseHolderReplicas := make(chan roachpb.ReplicaDescriptor, len(replicas))
```

Comment:
Looks like it is only accessed on lines 1161 and 1207. Am I misreading the code?